aten/src/ATen/native/Convolution.cpp

```diff
@@ -651,6 +651,15 @@ at::Tensor _convolution(
                           params.output_padding, params.stride, params.dilation,
                           params.groups);
     }
+    if (input_is_mkldnn && weight.is_mkldnn()) {
+      // mkldnn will error on the below 0-dim handling code
+      return empty_mkldnn(
```

Comment:
If support for empty binary ops is added in the previous PR, why does the code below still error out for mkldnn?
This is wrong, because with empty_mkldnn you are returning non-differentiable output.

Reply:
Should we revert this ?

Reply:
Why does it fail in the subsequent code? Is it the view ops? To preserve gradient flow, your output needs to be connected to weight and bias, so all you really need to do is be able to multiply and add with empty input, then the output has the correct history.